### PR TITLE
Temporary workaround for shotObstacle

### PR DESCRIPTION
**Pull request by ashaw596**
Only enables the shotObstacle during the kicking and aiming phase of pivot kick instead of the entire time for kicking. Just prevents the constant impossible paths when most of the screen is blocked off while the robot isn't even near the ball while capturing.

Should be replaced with #875

### CODE REVIEW

**Comment by jgkamat** [comment]
I'm interested in why the diff is so big, but I'll figure this out when I take a look at this.

I'll try to get to this this weekend, I'm slightly backed up right now, sorry! :cry:

**Comment by ashaw596** [comment]
Haha woops. One sec

**Comment by ashaw596** [comment]
Ok there you go. 3 lines.

File: soccer/gameplay/skills/pivot_kick.py [hunk 1]
```diff
@@ -111,6 +111,7 @@ def remove_aim_behavior(self):
             self.remove_subbehavior('aim')
 
     def on_enter_capturing(self):
+        self.enable_shot_obstacle = False
```
Comment (jgkamat) [inline soccer/gameplay/skills/pivot_kick.py]: This is slightly vauge (why its being done).
Could you add a comment saying something like:
"If kick fails, disable shot obstacle for second try"

**Pull request merged**